src/pages/index.jsx

```diff
@@ -18,15 +18,27 @@ export default function Home() {
               cheatsheets based on WCAG and ARIA best practices.
               Check back often as we’re adding new components frequently!
             </p>
-            <ul>
-              <li><Link href="/accordion"><a>Accordion</a></Link></li>
-              <li><Link href="/dialog"><a>Dialog</a></Link></li>
-              <li><Link href="/disclosure"><a>Disclosure</a></Link></li>
-              <li><Link href="/tabs"><a>Tabs</a></Link></li>
-            </ul>
+            <img
+              className="home-top__top-image"
+              src="/home-header-image.svg"
+              alt="Illustration of dog surrounded by accessible components"
+            />
           </div>
         </div>
-        <img className="home-top__top-image" src="/home-header-image.svg" alt="Illustration of dog surrounded by accessible components" />
+        <ul>
```

Comment:
I noticed that there's bullet points that show up because these are all a part of a list. If you want to remove them now, you can add the style `list-style: none;` to the list.

Reply:
I was worried about the screen reader not catching these, but it seems to continue to read off the text. How does it look? 
https://github.com/sparkbox/accessible-components/pull/25/commits/cc763ad17a1157163a1f3bae3529867080753548

Reply:
🏆 Looks good